src/Features/CSharp/Portable/Completion/CompletionProviders/DeclarationName/DeclarationNameInfo.cs

```diff
@@ -39,15 +42,15 @@ namespace Microsoft.CodeAnalysis.CSharp.Completion.Providers.DeclarationName
             IAliasSymbol? alias = null,
             ISymbol? symbol = null)
         {
-            PossibleSymbolKinds = possibleSymbolKinds;
+            _possibleSymbolKinds = possibleSymbolKinds;
             DeclaredAccessibility = accessibility;
             Modifiers = declarationModifiers;
             Type = type;
             Alias = alias;
             Symbol = symbol;
         }
 
-        internal static async Task<NameDeclarationInfo> GetDeclarationInfoAsync(Document document, int position, CancellationToken cancellationToken)
+        public static async Task<NameDeclarationInfo> GetDeclarationInfoAsync(Document document, int position, CancellationToken cancellationToken)
```

Comment:
this seems unrelated. I'm guessing just cleanup because the type is internal?

Reply:
yup :)